projects/admin/src/app/circulation/item/item.component.ts

```diff
@@ -53,34 +51,39 @@ export class ItemComponent implements OnInit {
   constructor(
     private recordService: RecordService,
     private organisationService: OrganisationService
-    ) { }
+    ) {  }
 
   ngOnInit() {
     if (this.item && this.item.loan && this.item.loan.pid) {
       const loanPid = this.item.loan.pid;
 
-      this.recordService.getRecords('fees', `loan.pid:${loanPid} AND fee_type:overdue`, 1, RecordService.MAX_REST_RESULTS_SIZE).pipe(
-        map((results: any) => results.hits.hits)
-      ).subscribe(fees => this.getTotalAmountOfFees(fees));
+      this.recordService.getRecords(
+        'fees', `loan.pid:${loanPid} AND fee_type:overdue`, 1, RecordService.MAX_REST_RESULTS_SIZE).pipe(
+        map((results: any) => results.hits.hits),
+        map((fees: any) => this._getTotalAmountOfFees(fees))
+      ).subscribe(total => this.totalAmountOfFee = total);
 
-      this.recordService.getRecords('notifications', `loan.pid:${loanPid}`, 1, RecordService.MAX_REST_RESULTS_SIZE).pipe(
+      this.notifications$ = this.recordService.getRecords(
```

Comment:
Not sure : `unsuscribe` ?

Reply:
No, it's not mandatory to unsubscribe because method resolves only one value and complete after.